Celigo.SuiteTalk/src/SuiteTalk/MissingReferences.cs

```diff
@@ -0,0 +1,154 @@
+namespace SuiteTalk
+{
+    public enum TransactionStatus
```

Comment:
How about we create a folder named `MissingReferneces` and create a `.cs` file for each of the types we are adding?